egs/swbd/ASR/RESULTS.md

```diff
@@ -0,0 +1,113 @@
+## Results
+### Switchboard BPE training results (Conformer-CTC)
+
+#### 2023-09-04
+
+The best WER, as of 2023-09-04, for the Switchboard is below
+
+Results using attention decoder is given as:
+
+|                                |  eval2000-swbd  |  eval2000-callhome  | eval2000-avg |
+|--------------------------------|-----------------|---------------------|--------------|
+|         `conformer_ctc`        |      9.48       |         17.73       |    13.67     | 
+
+Decoding results and models can be found here:
+https://huggingface.co/zrjin/icefall-asr-swbd-conformer-ctc-2023-8-26
```

Comment:
Please also upload the following files:
- the last checkpoint, e.g., epoch-100.pt
- the script for exporting the models
- a folder `test_wavs` containing some test waves for this model
- a README.md containing a link to this pull-request in icefall